VAMobile/src/api/appointments/getAppointments.tsx

```diff
@@ -36,8 +39,13 @@ export const useAppointments = (
   page: number,
   options?: { enabled?: boolean },
 ) => {
+  const { data: authorizedServices } = useAuthorizedServices()
+  const appointmentsInDowntime = useDowntime('appointments')
```

Comment:
Unfortunately, using `DowntimeFeatureTypeConstants` here caused unit tests to fail, because they don't recognize the constant. This was also [pointed out](https://github.com/department-of-veterans-affairs/va-mobile-app/blob/6634fd15d77afdb001620839e67e1acdb784711e/VAMobile/src/screens/SyncScreen/SyncScreen.tsx#L40) a while ago. We'll need to look into this